clang/lib/Basic/Targets/AArch64.cpp

```diff
@@ -860,7 +841,7 @@ bool AArch64TargetInfo::handleTargetFeatures(std::vector<std::string> &Features,
     }
     if (Feature == "+dit")
       HasDIT = true;
-    if (Feature == "+cccp")
+    if (Feature == "+ccpp")
```

Comment:
Again, does this need a test?

Reply:
Agree, but not yet sure how it is better to test this